pkg/image/qemu.go

```diff
@@ -226,18 +226,18 @@ func checkIfURLIsValid(info *ImgInfo, availableSize int64, filesystemOverhead fl
 		}
 	}
 
-	if int64(float64(availableSize)*(1-filesystemOverhead)) < info.VirtualSize {
-		return errors.Errorf("Virtual image size %d is larger than available size %d (PVC size %d, reserved overhead %f%%). A larger PVC is required.", info.VirtualSize, int64((1-filesystemOverhead)*float64(availableSize)), info.VirtualSize, filesystemOverhead)
+	if availableSize < info.VirtualSize {
+		return errors.Errorf("Virtual image size %d is larger than the reported available storage %d (PVC size %d). A larger PVC is required.", info.VirtualSize, availableSize, info.VirtualSize)
```

Comment:
I am not sure it is PVC size that we report here. Isn't it a disk or an image size?

Reply:
Yeah, PVC size here is definitely the image's virtual size and can confuse,
we do however print the calculated PVC size in https://github.com/kubevirt/containerized-data-importer/blob/654c4cdd11101ad158129c468873581569debdad/pkg/importer/data-processor.go#L404